### DISCUSSION TITLE
TG-FileStreamBot 3.0.0-alpha2 is out

### DISCUSSION THREAD

**Discussion by EverythingSuckz** [EverythingSuckz/TG-FileStreamBot#129] [Announcements]
☁️ [File stream bot [golang] version 3.0.0-alpha2](https://github.com/EverythingSuckz/TG-FileStreamBot/releases/tag/3.0.0-alpha2) is out

🆕 What's new?

- 👤 User session support
   Now it's easier than ever to add the bots to channel as admin automatically.

- ⚡️ Changed starting behavior
Now you can use the command to start the bot.
```bash
./fsb run
```

⚙️ String session generator
   New pyrogram session generator using QR code login was built into the binary.
Run the command for more info
```bash
./fsb session --help
```

- 🔄 Add flood retry middleware
 This will wait and retry the request if there's a flood wait error.

- 🔧 Other minor bug fixes and features

Please report bugs in the discussion chat or in the GitHub issues.

**Comment by 5bc2e7** [comment]
It would be nice to support multi-threaded downloads like this project does: [https://github.com/iyear/tdl](https://github.com/iyear/tdl)
